sale_commission/tests/test_sale_commission.py

```diff
@@ -285,6 +285,7 @@ def test_sale_commission_net_amount_invoice(self):
         wizard2 = self.make_inv_model.create({'product': 1})
         wizard2.button_create()
         settlements = self.settle_model.search([('state', '=', 'invoiced')])
+        self.assertGreater(len(settlements), 0)
```

Comment:
You can do: `self.assertTrue(settlements)`

Reply:
Much cleaner, done